Updated to include Account and team setup events, system console events (#3518)
mattermost/docs: source/administration/telemetry.rst

Non-personally Identifiable Diagnostic Information, distinguished by end users and System Admins
  Boolean when the following events occur:

>   - *Team and Account Setup Diagnostics:* Account creation via email, invite or UI, account creation page view, account creation completion; tutorial step and tip completion or opt out, team creation page view, team name and URL entry, team creation completion. Click events on signup_email, select_team, display_name and team_url pages.

Reviewer: The events are actually already included
 - Account creation via email
 - team creation page view, team name and URL entry, team creation completion

We can add `select_team` event though. So perhaps:

Reviewer: ```suggestion
  - *Team and Account Setup Diagnostics:* Account creation via email, invite or UI, account creation page view, account creation completion; tutorial step and tip completion or opt out, team creation page view, team name and URL entry, team creation completion, team selection page.
```

Author: Well the change is to track clicks on all form elements, buttons, textboxes and links on these pages, not just the page views.

Reviewer: ```suggestion
  - *Team and Account Setup Diagnostics:* Account creation via email, invite or UI, account creation page view, account creation completion; tutorial step and tip completion or opt out, team creation page view, team name and URL entry, team creation completion, clicks on all form elements, buttons, textboxes and links on sign up page, team selection page and team creation pages
```

Reviewer: Got it, how about this?

Author: yep wfm

  - *Mobile Performance Diagnostics:* Load times for starting the app, switching channels, and switching teams 
  - *Permissions Discovery Diagnostics (Enterprise Edition Only):* Provides all the permissions configured for each role for the System Scheme and each Team Override Scheme created in the system.  Scheme ID; Team Admin Permissions; Team User Permissions; Channel Admin Permissions; Channel User Permissions; Number of teams the scheme is associated with
  - *Group Discovery Diagnostics (Enterprise Edition Only):* Provides information related to AD/LDAP groups, including number of groups synced to Mattermost, teams and channels associated to groups, teams and channels synced with groups, and number of group members

>   - *System Console Discovery Diagnostics:* click events on the hamburger menu and menu items: Administrator's Guide, Troubleshooting Forum, Commerical Support, About Mattermost; click events on the menu itesm in the left navigation.

Reviewer: Is this for discovering the System Console? Or is this just for the menu items?

Author: ```suggestion
  - *System Console Discovery Diagnostics:* click events on the hamburger menu and menu items: Administrator's Guide, Troubleshooting Forum, Commerical Support, About Mattermost; click events on the menu items in the left navigation.
```

Author: Just for the menu items

Reviewer: Just minor tweak:

Reviewer: ```suggestion
  - *System Console Menu Discovery Diagnostics:* Clicks on the hamburger menu items of the System Console, including Administrator's Guide, Troubleshooting Forum, Commercial Support, About Mattermost, and clicks on the left-hand side navigation menu items
```
